src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Circle");
+        System.out.println("Color: " + getColor());
+        System.out.println("Radius: " + radius);
+        System.out.println("Area: " + getArea());
```

Comment:
The draw() method is combining two different responsibilities: displaying the figure type and calculating the area. Consider separating concerns by moving the area calculation out of the draw() method.